src/front-door/azext_front_door/_params.py

```diff
@@ -114,6 +127,12 @@ def load_arguments(self, _):
         c.argument('custom_forwarding_path', help='Custom path used to rewrite resource paths matched by this rule. Leave empty to use incoming path.')
         c.argument('dynamic_compression', arg_type=get_three_state_flag(positive_label='Enabled', negative_label='Disabled', return_label=True), help='Use dynamic compression for cached content.')
         c.argument('query_parameter_strip_directive', arg_type=get_enum_type(FrontDoorQuery), help='Treatment of URL query terms when forming the cache key.')
+        c.argument('redirect_type', arg_type=get_enum_type(FrontDoorRedirectType), help='The redirect type the rule will use when redirecting traffic. Default: Moved.')
```

Comment:
You shouldn't need to spell out the default here. The CLI will automatically pick up the default from the method signature if it is available. If you are calling out the service default (i.e. if None is sent to the service) then this may become inaccurate.

If you actually want to *specify* the default, then use the `default=...` kwarg, which will automatically populate this help text and ensure there's never a difference between what the help says and the CLI sends.

Reply:
Fixed